src/k8sSettings.json

```diff
@@ -102,29 +135,43 @@
                         "type": "object"
                       },
                       "requestParamsSecret": {
-                        "description": "RequestParamsSecret is a reference to a secret for additional parameters you'd prefer not to provide in plaintext",
+                        "description": "RequestParamsSecret is a reference to a secret for additional parameters you'd prefer not to provide in plaintext.",
                         "type": "string"
                       },
                       "scopes": {
-                        "description": "Scopes see https://www.pomerium.com/reference/#identity-provider-scopes",
+                        "description": "Scopes see https://www.pomerium.com/reference/#identity-provider-scopes.",
                         "items": {
                           "type": "string"
                         },
                         "type": "array"
                       },
                       "secret": {
-                        "description": "Secret refers to a k8s secret containing IdP provider specific parameters and must contain at least `client_id` and `client_secret` map values",
+                        "description": "Secret containing IdP provider specific parameters and must contain at least client_id and client_secret values, an optional `service_account` field, mapped to https://www.pomerium.com/reference/#identity-provider-service-account",
```

Comment:
As the link is, it redirects to reference homepage. I tried correcting it:  https://www.pomerium.com/reference/identity-provider-service-account

But, it's 404ing